src/main/java/net/snowflake/client/jdbc/SnowflakeBaseResultSet.java

```diff
@@ -1384,4 +1470,13 @@ public boolean isWrapperFor(Class<?> iface) throws SQLException {
 
     return iface.isInstance(this);
   }
+
+  private BiConsumer<SQLData, SQLInput> readSqlDataConsumer =
+      (SQLData instance, SQLInput sqlInput) -> {
+        try {
+          instance.readSQL(sqlInput, null);
+        } catch (SQLException e) {
+          throw new RuntimeException(e);
```

Comment:
No remapping should happen. We shouldn't remap exceptions only because Java's limitations in lambdas.

Reply:
Fixed